CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## vNEXT (not yet published)
 
+### `@liveblocks/emails`
+
+- Revert the deduplication logic in email notification preparation functions
```

Comment:
I don't think we need to repeat the exact names since this package doesn't contain other things, but we could mention the version of the last change 

```md
- Revert deduplication logic introduced in `v2.18.0` as it provided no measurable benefits while increasing complexity.
```